ingestion/src/metadata/ingestion/processor/pii.py

```diff
@@ -57,25 +60,107 @@ class PiiTypes(Enum):
     TAX_ID = auto()
     KEY = auto()
     BANKACC = auto()
+    ORGANIZATION = auto()
+    DATE_TIME = auto()
 
 
 class TagType(Enum):
     SENSITIVE = "Sensitive"
     NONSENSITIVE = "NonSensitive"
 
 
-class ColumnPIIType(BaseModel):
-    pii_types: PiiTypes
-    tag_type: TagType
+class NERScanner:
+    """A scanner that uses Spacy NER for entity recognition"""
+
+    def __init__(self):
+        self.nlp = spacy.load("en_core_web_sm")
+
+    def get_spacy_to_pii_type(self, spacy_type):
+        spacy_to_pii = {
+            "ORG": PiiTypes.ORGANIZATION.value,
+            "GPE": PiiTypes.LOCATION.value,
+            "TIME": PiiTypes.DATE_TIME.value,
+            "CARDINAL": PiiTypes.UNSUPPORTED.value,
+            "PERSON": PiiTypes.PERSON.value,
+            "FAC": PiiTypes.LOCATION.value,
+            "LOC": PiiTypes.LOCATION.value,
+        }
```

Comment:
let's have this available globally, use a specific method to just fetch the data from enum

Reply:
OK 👍